client/oci_env/utils.py

```diff
@@ -71,6 +71,9 @@ def get_config(env_file):
         "API_PROTOCOL": "http",
         "API_HOST": "localhost",
         "API_PORT": "5001",
+
+        # A port dedicated for exposing generated docs
+        "DOCS_PORT": "12345",
```

Comment:
@lubosmj I had to revert this it was breaking ci because the test profiles bother have the same DOCS_PORT, which was causing it to fail when you spin it up twice.